mkstage4.sh

```diff
@@ -16,6 +16,42 @@ USER_EXCL=""
 S_KERNEL=0
 x86_64=0
 PARALLEL=0
+HAS_PORTAGEQ=0
+
+# Excludes - whitespace delimited list of things to leave out
+EXCLUDES_LIST="
+ home/*/.bash_history\
+ dev\
+ var/tmp\
+ media\
+ mnt\
+ proc\
+ run\
+ sys\
+ tmp\
+ var/lock\
+ var/log\
+ var/run\
+ var/lib/docker"
+
+EXCLUDES_LIST_PORTAGE="
+ var/db/repos/gentoo/*\
+ usr/portage*\
+ var/cache/distfiles/*"
+ 
+# Excludes function - create tar --exclude=foo options
+exclude()
+{
+  ADDEXCLUDE=$(echo "$1" | sed 's/^\///')
+  EXCLUDES+=" --exclude=${TARGET}${ADDEXCLUDE}"
+}
+
+# Check if portageq is available
+if [ `which portageq` ]
```

Comment:
Please move this bit of logic (and the subsequent invocations of `portageq`) to the [previous pull request](https://github.com/TheChymera/mkstage4/pull/18), observing the conditionality I recommended [here](https://github.com/TheChymera/mkstage4/pull/17#issuecomment-517961197).